src/Nancy/HttpLinkBuilder.cs

```diff
@@ -0,0 +1,49 @@
+namespace Nancy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Builds an RFC 5899 Link HTTP header as a <see cref="List{T}" /> of <see cref="HttpLink" /> objects.
+    /// </summary>
+    /// <seealso cref="HttpLink" />
+    public class HttpLinkBuilder : List<HttpLink>
```

Comment:
Again, yes, I agree, but it also provides a lot of other methods that might not make sense. We shouldn't inherit from `List<T>` just because of laziness :smile: